### PR TITLE
Fix 2 quicktour file doctest

### PR DESCRIPTION
**Pull request by ydshieh**
# What does this PR do?

Update expect output values due to some update of package or Hub repo file.

### CODE REVIEW

File: docs/source/en/quicktour.mdx [hunk 1]
```diff
@@ -118,7 +118,7 @@ Extract the raw waveform arrays from the first 4 samples and pass it as a list t
 ```py
 >>> result = speech_recognizer(dataset[:4]["audio"])
 >>> print([d["text"] for d in result])
-['I WOULD LIKE TO SET UP A JOINT ACCOUNT WITH MY PARTNER HOW DO I PROCEED WITH DOING THAT', "FODING HOW I'D SET UP A JOIN TO HET WITH MY WIFE AND WHERE THE AP MIGHT BE", "I I'D LIKE TOY SET UP A JOINT ACCOUNT WITH MY PARTNER I'M NOT SEEING THE OPTION TO DO IT ON THE AP SO I CALLED IN TO GET SOME HELP CAN I JUST DO IT OVER THE PHONE WITH YOU AND GIVE YOU THE INFORMATION OR SHOULD I DO IT IN THE AP AND I'M MISSING SOMETHING UQUETTE HAD PREFERRED TO JUST DO IT OVER THE PHONE OF POSSIBLE THINGS", 'HOW DO I THURN A JOIN A COUNT']
+['I WOULD LIKE TO SET UP A JOINT ACCOUNT WITH MY PARTNER HOW DO I PROCEED WITH DOING THAT', "FONDERING HOW I'D SET UP A JOIN TO HELL T WITH MY WIFE AND WHERE THE AP MIGHT BE", "I I'D LIKE TOY SET UP A JOINT ACCOUNT WITH MY PARTNER I'M NOT SEEING THE OPTION TO DO IT ON THE APSO I CALLED IN TO GET SOME HELP CAN I JUST DO IT OVER THE PHONE WITH YOU AND GIVE YOU THE INFORMATION OR SHOULD I DO IT IN THE AP AN I'M MISSING SOMETHING UQUETTE HAD PREFERRED TO JUST DO IT OVER THE PHONE OF POSSIBLE THINGS", 'HOW DO I FURN A JOINA COUT']
```
Comment (ydshieh) [inline docs/source/en/quicktour.mdx:121]: `librosa` has new version `0.10.0` (previously `0.9.2`) 2 days ago, and it changes a bit the values produced by some audio datasets.

File: docs/source/es/quicktour.mdx [hunk 2]
```diff
@@ -129,7 +129,7 @@ Extraigamos las matrices de onda cruda (raw waveform, en inglés) de las primera
...
+['ahora buenas eh a ver tengo un problema con vuestra aplicación resulta que que quiero hacer una transferencia bancaria a una cuenta conocida pero me da error la aplicación a ver que a ver que puede ser', 'la aplicación no cargue saldo de mi nueva cuenta', 'hola tengo un problema con la aplicación no carga y y tampoco veo que carga el saldo de mi cuenta nueva dice que la aplicación está siendo reparada y ahora no puedo acceder a mi cuenta no necesito inmediatamente', 'hora buena la aplicación no se carga la vida no carga el saldo de mi cuenta nueva dice que la villadenta siendo reparada y oro no puedo hacer a mi cuenta']
```
Comment (ydshieh) [inline docs/source/es/quicktour.mdx:132]: `librosa`

File: docs/source/es/quicktour.mdx [hunk 3]
```diff
@@ -87,7 +87,7 @@ El pipeline descarga y almacena en caché el [modelo preentrenado](https://huggi
 
 ```py
 >>> clasificador("Estamos muy felices de mostrarte la biblioteca de 🤗 Transformers.")
-[{'label': 'POS', 'score': 0.9916}]
+[{'label': 'POS', 'score': 0.9320}]
```
Comment (ydshieh) [inline docs/source/es/quicktour.mdx:90]: The used reop. `pysentimiento/robertuito-sentiment-analysis` has some recent updates on modeling/config/tokenizer files.

**Review by MKhalusova — APPROVED** [review]
Thank you for updating these!

**Review by sgugger — APPROVED** [review]
Thanks a lot for fixing this!

**Pull request merged**